*** md.h	Fri Oct  2 02:46:43 2009
--- ../st-1.9-patch/md.h	Mon Mar 16 15:11:49 2015
***************
*** 422,428 ****
  #define MD_STACK_GROWS_DOWN
  
  #if defined(__GLIBC__) && __GLIBC__ >= 2
! #define MD_GET_SP(_t) (_t)->context[0].__jmpbuf[20]
  #else
  #error "ARM/Linux pre-glibc2 not supported yet"
  #endif /* defined(__GLIBC__) && __GLIBC__ >= 2 */
--- 422,428 ----
  #define MD_STACK_GROWS_DOWN
  
  #if defined(__GLIBC__) && __GLIBC__ >= 2
! #define MD_GET_SP(_t) (_t)->context[0].__jmpbuf[8]
  #else
  #error "ARM/Linux pre-glibc2 not supported yet"
  #endif /* defined(__GLIBC__) && __GLIBC__ >= 2 */
