Only in .: 1.st.arm.patch
diff -r -c ./md.h ../st-1.9-patch-arm/md.h
*** ./md.h	2009-10-02 02:46:43.000000000 +0800
--- ../st-1.9-patch-arm/md.h	2014-03-16 20:49:03.845344804 +0800
***************
*** 422,428 ****
  #define MD_STACK_GROWS_DOWN
  
  #if defined(__GLIBC__) && __GLIBC__ >= 2
! #define MD_GET_SP(_t) (_t)->context[0].__jmpbuf[20]
  #else
  #error "ARM/Linux pre-glibc2 not supported yet"
  #endif /* defined(__GLIBC__) && __GLIBC__ >= 2 */
--- 422,428 ----
  #define MD_STACK_GROWS_DOWN
  
  #if defined(__GLIBC__) && __GLIBC__ >= 2
! #define MD_GET_SP(_t) (_t)->context[0].__jmpbuf[8]
  #else
  #error "ARM/Linux pre-glibc2 not supported yet"
  #endif /* defined(__GLIBC__) && __GLIBC__ >= 2 */
